tests/ledctl/ledctl_cmd.py

```diff
@@ -70,6 +70,16 @@ def run_ledctl_cmd_valid(self, params: list):
         LOGGER.debug(f"Command returned:\n {out}")
         return out
 
+    # Run ledctl command and expect it to fail
+    def run_ledctl_cmd_not_valid(self, params: list):
+        result = self.run_ledctl_cmd(params, output=True)
+        if result.returncode == 0:
+            raise Exception("Command succeed, but was expected to fail!")
+        out = result.stderr
```

Comment:
can we return result for both valid/invalid?
I can realize that in the future we may add tests where few messages are logged to stdout, and later error is raised and we should abort but checking stdout first would be crucial to check the command flow.